eda-env/Lib/site-packages/pandas/tests/window/test_numba.py

```
@@ -0,0 +1,465 @@
+
+        roll = data.rolling(3, step=step)
+        result = getattr(roll, method)(
+            engine="numba", engine_kwargs=engine_kwargs, **kwargs
+        )
+        expected = getattr(roll, method)(engine="cython", **kwargs)
+        tm.assert_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "data", [DataFrame(np.eye(5)), Series(range(5), name="foo")]
+    )
+    def test_numba_vs_cython_expanding_methods(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)